util/stmtsummary/statement_summary.go

```diff
@@ -572,6 +574,7 @@ func newStmtSummaryByDigestElement(sei *StmtExecInfo, beginTime int64, intervalS
 	ssElement := &stmtSummaryByDigestElement{
 		beginTime: beginTime,
 		sampleSQL: formatSQL(sei.OriginalSQL),
+		hints:     sei.Hints,
```

Comment:
Only record the first hint of this kind of SQL?
For example, you execute 2 statements, one with a hint and one without a hint. Then what's the expected result?

Reply:
This hint is recovered from the current execution plan. If we the execution plan can be transformed into a hint string, then we store it.

Reply:
I don't think you understood me.
stmt1: `select * from t`.
stmt2: `/*some hint*/ select * from t`.
The digest and plan digest of stmt1 and stmt2 are the same. 
In your implementation, after executing these 2 statements, the hint is not recorded, because you only record the first hint for each statement. Is that expected?